### DISCUSSION TITLE
The trace list does not refresh after clicking the refresh button

### DISCUSSION THREAD

**Discussion by BertOnline** [apache/skywalking#9709] [General]
Click the Refresh button, and the trace list will not be refreshed, resulting in the inability to view the latest trace log. Is it specially designed like this?

![image](https://user-images.githubusercontent.com/17882970/192933789-fab48701-6e37-40d7-9e31-41ddb2dd32a4.png)

**Comment by wu-sheng** [comment]
Yes, only `search` button works on this page.
